docs/modding/README.md

```diff
@@ -1,5 +1,6 @@
 # Modding
 
-The documentation for modding has been moved over to [r2northstar.readthedocs.io](https://r2northstar.readthedocs.io)
+### Modding documentation for Northstar and Titanfall 2 has moved over to: [https://r2northstar.readthedocs.io/](https://r2northstar.readthedocs.io/).
```

Comment:
I'd keep it without the header formatting btw. Also technically it's just for Northstar so:

```suggestion
Modding documentation for Northstar has moved over to: [https://r2northstar.readthedocs.io/](https://r2northstar.readthedocs.io/).
```

Reply:
Fixed.